src/main/java/io/kestra/plugin/meilisearch/DocumentAdd.java

```diff
@@ -0,0 +1,85 @@
+package io.kestra.plugin.meilisearch;
+
+import com.meilisearch.sdk.Client;
+import com.meilisearch.sdk.Index;
+import com.meilisearch.sdk.exceptions.MeilisearchException;
+import com.meilisearch.sdk.model.TaskInfo;
+import com.meilisearch.sdk.model.TaskStatus;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.slf4j.Logger;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Add Document",
+    description = "Add Document to Meilisearch"
+)
+@Plugin(
+    examples = {
+        @io.kestra.core.models.annotations.Example(
+            title = "Add Document to Meilisearch",
+            code = {
+                """
+                    document: \\"document to be added in JSON\\",
+                    url: \\"url of the meilisearch server\\",
+                    key: \\"masterKey of the meilisearch server\\",
+                    index: \\"index where we want to add the document\\"
```

Comment:
```suggestion
                """
                    document: "document to be added in JSON"
                    URL:  "url of the meilisearch server"
                    key:  "masterKey of the meilisearch server"
                    index: "index where we want to add the document"
```

Java text block allow to use double-quotes inside them.
Moreover, we usually provide a "working example" not an example with description of the properties.

Reply:
Added explicit flows